jujugui/static/gui/src/app/jujulib/test-payment.js

```diff
@@ -586,4 +591,238 @@ describe('jujulib payment service', function() {
         });
     });
   });
+
+  describe('addAddress', () => {
+    let address;
+
+    beforeEach(() => {
+      address = {
+        name: 'Home',
+        line1: '1 Maple St',
+        line2: null,
+        county: 'Bunnyhug',
+        city: 'Sasquatch',
+        postcode: '90210',
+        countryCode: 'CA'
+      };
+    });
+
+    it('can add an address', () => {
+      const originalMakeRequest = jujulib._makeRequest;
+      const makeRequest = sinon.stub();
+      jujulib._makeRequest = makeRequest;
```

Comment:
I believe you can do this with:
```
const makeRequest = sinon.stub(jujulib, '_makeRequest');
```
And then clean it up later with `makeRequest.restore()`.

It looks like you've used this pattern in most of your tests, so you could redo them all like this.